src/com/rcv/ResultsWriter.java

```diff
@@ -44,6 +45,8 @@
   private Map<Integer, List<String>> roundToWinningCandidates;
   // configuration file in use for this election
   private ElectionConfig config;
+  // timestamp string to use when generating output file names
+  private String timestamp;
```

Comment:
Might be more clear to call this timestampString.

Reply:
fixed